### ISSUE TITLE
import nats fails after pip install nats-client

### ISSUE THREAD

**Issue by digitalnomd** [nats-io/python-nats#31]
After installing with pip install nats-client, I am getting the following error when I try to import. 

```
Python 2.7.12 (default, Dec  4 2017, 14:50:18)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import nats
Traceback (most recent call last):
File "<stdin>", line 1, in <module>
File "nats.py", line 6, in <module>
from nats.io.utils import new_inbox
ImportError: No module named io.utils
```

This is weird because it was importing correctly before. 

I also checked the nats package that pip installed and there is a nats.io.utils file so maybe there is something going on with the packaging.

**Comment by wallyqs** [comment]
Thanks for the report, can't reproduce on my env but will look into simplifying the packaging...

**Comment by wallyqs** [comment]
Fixed via #34
